packages/package.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
@takeutak Does this file belong to a specific package or does it have some other purpose (because it's not under any of the package sub-folders but rather in the ./packages/ directory itself)